src/custom/constants/index.ts

```diff
@@ -1,7 +1,6 @@
 import { ChainId } from '@uniswap/sdk'
 
-// TODO: Load app Id
-export const APP_ID = 0
+export const APP_ID = Number(process.env.REACT_APP_ID || 0)
```

Comment:
dont need 0 in number constructor

Reply:
Good for me, I will change. I thought it was more explicit